packages/webview_flutter/webview_flutter_web/lib/shims/dart_ui_fake.dart

```diff
@@ -0,0 +1,30 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:html' as html;
+
+// ignore_for_file: camel_case_types, avoid_classes_with_only_static_members
+
+// Fake interface for the logic that this package needs from (web-only) dart:ui.
+// This is conditionally exported so the analyzer sees these methods as available.
+
+/// Shim for web_ui engine.PlatformViewRegistry
+/// https://github.com/flutter/engine/blob/master/lib/web_ui/lib/ui.dart#L62
+class platformViewRegistry {
+  /// Shim for registerViewFactory
+  /// https://github.com/flutter/engine/blob/master/lib/web_ui/lib/ui.dart#L72
+  static registerViewFactory(
+      String viewTypeId, html.Element Function(int viewId) viewFactory) {}
+}
+
+/// Shim for web_ui engine.AssetManager.
+/// https://github.com/flutter/engine/blob/master/lib/web_ui/lib/src/engine/assets.dart#L12
+class webOnlyAssetManager {
+  /// Shim for getAssetUrl.
+  /// https://github.com/flutter/engine/blob/master/lib/web_ui/lib/src/engine/assets.dart#L45
+  static getAssetUrl(String asset) {}
+}
```

Comment:
See https://github.com/flutter/plugins/blob/2ab3044cfbb4ee2f3dcbc5741628da8f4e0eb437/packages/url_launcher/url_launcher_web/lib/src/shims/dart_ui_fake.dart#L15-L30 for a version of these that passes analysis; I just fixed this for url_launcher.

Reply:
Ah thank you, that seems to do the trick.